src/blocks/carousel/carousel.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * BLOCK - Renders a Carousel
+ *
+ * @link https://codex.wordpress.org/Template_Hierarchy
+ *
+ * @package abs
+ */
+
+use function WebDevStudios\abs\print_module;
+use function WebDevStudios\abs\print_element;
+use function WebDevStudios\abs\get_formatted_atts;
+use function WebDevStudios\abs\get_acf_fields;
+use function WebDevStudios\abs\get_block_classes;
+
+$abs_defaults = [
+	'class'               => [ 'wds-block', 'carousel' ],
+	'show_arrows'         => true,
+	'show_pagination'     => true,
+	'allowed_innerblocks' => [ 'core/heading', 'core/paragraph' ],
+	'id'                  => ! empty( $block['anchor'] ) ? $block['anchor'] : '',
+];
+
+
+// Get custom classes for the block and/or for block colors.
+$abs_block_classes = [];
+$abs_block_classes = get_block_classes( $block );
+
+if ( ! empty( $abs_block_classes ) ) :
+	$abs_defaults['class'] = array_merge( $abs_defaults['class'], $abs_block_classes );
+endif;
+
+// Set up element attributes.
+$abs_atts = get_formatted_atts( [ 'class', 'id' ], $abs_defaults );
+
+// Pull in the fields from ACF.
+$abs_heros = get_acf_fields( [ 'overlay', 'hero' ], $block['id'] );
```

Comment:
Shouldn't this be labeled `$abs_carousel`? Also I think the field group should be `'carousel'`, no?